### ISSUE TITLE
Dynamodb cleanup via TTLs

### ISSUE THREAD

**Issue by sarcasticadmin** [chiradeep/lambda-mutex#5]
In regards to one of the limitations mentioned in the `README.md`:
```
No cleanup. Each named mutex gets a row in the Dynamodb table. These rows are never cleaned up
even if nobody is using the mutex. It should be trivial to write a lambda to vaccuum these
rows, however, given the small size of the row and the infinite capacity of DynamoDb, it is not
likely to be a problem, if used for its intended purpose.
```

I was wondering if you had any thoughts regarding enabling `TTL`s on the mutex dynamodb table? Checkout the announcement regarding `TTL`s from last year [here](https://aws.amazon.com/about-aws/whats-new/2017/02/amazon-dynamodb-now-supports-automatic-item-expiration-with-time-to-live-ttl/) and an example of enabling it on the table [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html)

You could add an additional attribute, something like `ttl`, to set the expiration of entries instead of trying to leverage `expire_ts` since users might still want to be able to view table entries for a given period of time after lock expiration. But overall it should be possible to get cleanup of old locks with minimal additional configuration.

Let me know what you think!

**Comment by chiradeep** [comment]
Looks neat. I can take a stab at it but it might be some time since I am away from a computer for a few weeks. PR welcome!

**Comment by sarcasticadmin** [comment]
awesome, thanks for getting back to me @chiradeep! I'll leave it unassigned for now, hopefully I can get around to putting together a PR for this sometime soon.

**Comment by chiradeep** [comment]
Closed by #7
